web/src/components/tasks/business-formation/getErrorStateForField.ts

```diff
@@ -52,7 +67,6 @@ export const getErrorStateForField = (
     "canMakeDistribution",
     "addressState",
     "addressCountry",
-    "addressMunicipality",
```

Comment:
Why are you removing this? addressMunicipality is an object field, not a text field